README.md

```diff
@@ -49,7 +49,18 @@ sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.d/99-local.yaml
 vagrant up centos7-devel
 ```
 
-You can find more thorough guides in the [docs folder](https://github.com/theforeman/forklift/tree/master/docs).
+In case using vagrant is not desired, ansible playbooks and roles from this repo can be used separately. This is useful if existing host should be used for the installation, e.g. a beaker machine. In order to deploy devel environment on host test.example.com, following should be executed:
+
+```sh
+echo "[devel]\ntest.example.com" > local_inventory
+ansible-playbook --private-key=/home/ares/.ssh/id_rsa --user root --inventory local_inventory --extra-vars katello_devel_github_username=katello playbooks/devel.yml
```

Comment:
I'd replace your username with ```USERNAME``` or ```$HOME``` if that works.

Reply:
`~` works nice too.

Reply:
replaced with `~`